### ISSUE TITLE
How to use Docker from a windows machine

### ISSUE THREAD

**Issue by cansav09** [AlexsLemonade/RNA-Seq-Exercises#20]
If Kitematic doesn't work, how do we have them use Docker from command line? Do the same commands work as they do for Mac? 

Change repository to reflect this.

**Comment by jaclyn-taroni** [comment]
With the updates in #22, we have to set a password from the command line. So the steps will be (Mac and Linux at least):

1. Pull the appropriate image 
```
docker pull ccdl/training_rnaseq:2019-houston
```
2. Run the container
```
docker run -e PASSWORD=<PASSWORD> -p 8787:8787 ccdl/training_rnaseq:2019-houston
```
3. Open Kitematic - you should see an image running.
4. `Settings` > `Volumes` > Set local folder to "main directory" that was transferred from the flash/hard drive
5. (Optional) `Settings` > `General` > Can rename the container to something like `training_rnaseq` by changing the `NAME` field
6. Navigate to `localhost:8787` in your web browser
7. Log into RStudio. The username will be `rstudio` and the password will be whatever you selected above (can also be accessed from the `Settings` > `General` panel). 
8. You should see a `kitematic/` folder in your RStudio `Files` panel. Click on it. If you do not see the training modules folders in the kitematic folder, raise your hand.

**Comment by jaclyn-taroni** [comment]
Just gave this a try on the ALSF desktop (Windows 7 Professional). The commands worked. Navigating to `localhost:8787` does not. So we should probably say

6. Go to the `Home` tab and look for the `ACCESS URL` in the `IP & PORTS` pane. Copy this URL and paste it into the navigator bar of your web browser. 

This is similar to what we have here now: https://github.com/AlexsLemonade/RNA-Seq-Exercises#post-docker-steps

**Comment by cansav09** [comment]
Do we want to split this into a "Kitematic" and "without Kitematic" steps?

**Comment by jaclyn-taroni** [comment]
For now, I might do "Before Kitematic" (steps 1&2) and "With Kitematic" (3-8) and then see what tomorrow's dry run brings.

**Comment by cansav09** [comment]
The current steps 1 & 2 on the README are written as using Kitematic to pull the image. So I'm not sure how you mean to divvy this up.

**Comment by jaclyn-taroni** [comment]
We have to set a password from the command line. Steps 1 & 2 above replace what is currently on the README.

**Comment by cansav09** [comment]
Can we get rid of the Optional step 5? I feel like it's doesn't help much and only adds an extra step where something could get messed up.

**Comment by jaclyn-taroni** [comment]
Sure

**Comment by cansav09** [comment]
This was partially addressed by #24, but we need to do further testing with the Windows 10 computers that we should be using tomorrow.

**Comment by jaclyn-taroni** [comment]
Closed by https://github.com/AlexsLemonade/training-modules/pull/51
